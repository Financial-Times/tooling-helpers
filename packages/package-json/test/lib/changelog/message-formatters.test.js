const messageFormatters = require('../../../src/lib/changelog/message-formatters');

describe("handler", () => {

    test("throws an error if there is no message formatter defined for the entry's `event`", () => {
        expect(() => {
            messageFormatters.handler({
                event: 'invalidEvent'
            })
        }).toThrowErrorMatchingSnapshot();
    });

    test("throws an error if there is no message formatter defined for the entry's `event`", () => {
        expect(() => {
            messageFormatters.handler({
                event: 'invalidEvent'
            })
        }).toThrowErrorMatchingSnapshot();
    });

});

describe.skip("setField", () => {

    test("...", () => {

    });

});

describe.skip("requireDependency", () => {

    test("...", () => {

    });

});

describe.skip("removeDependency", () => {

    test("...", () => {

    });

});

describe.skip("requireScript", () => {

    test("...", () => {

    });

});
