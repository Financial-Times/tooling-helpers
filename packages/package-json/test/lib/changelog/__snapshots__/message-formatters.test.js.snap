// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler throws an error if there is no message formatter defined for the entry's \`event\` 1`] = `"message-formatters#handler: There is no message formatter defined for the \`event\` 'invalidEvent'"`;

exports[`handler throws an error if there is no message formatter defined for the entry's \`event\` 2`] = `"message-formatters#handler: There is no message formatter defined for the \`event\` 'invalidEvent'"`;
